Exiled.Events/EventArgs/Scp079/GainingExperienceEventArgs.cs

```diff
@@ -26,24 +26,34 @@ public class GainingExperienceEventArgs : IPlayerEvent, IDeniableEvent
         /// <param name="gainType">
         ///     <inheritdoc cref="GainType" />
         /// </param>
+        /// <param name="roleType">
+        ///     <inheritdoc cref="RoleType" />
+        /// </param>
         /// <param name="amount">
         ///     <inheritdoc cref="Amount" />
         /// </param>
         /// <param name="isAllowed">
         ///     <inheritdoc cref="IsAllowed" />
         /// </param>
-        public GainingExperienceEventArgs(Player player, Scp079HudTranslation gainType, int amount, bool isAllowed = true)
+        public GainingExperienceEventArgs(Player player, Scp079HudTranslation gainType, int amount, RoleTypeId roleType, bool isAllowed = true)
         {
             Player = player;
             GainType = gainType;
+            RoleType = roleType;
             Amount = amount;
             IsAllowed = isAllowed;
         }
 
         /// <summary>
-        ///     Gets the experience gain type.
+        ///     Gets or sets the from whish role you gain experience.
```

Comment:
```suggestion
        ///     Gets or sets the role that was used to gain experience.
```
Can't think of a much better description than this lol

Reply:
ewwww

Reply:
@louis1706 at least use Grammarly to do spell check and basic grammar check